services/galley/test/integration/API/MLS.hs

```diff
@@ -2276,3 +2149,38 @@ getGroupInfoDisabled = do
     withMLSDisabled $
       getGroupInfo (qUnqualified alice) qcnv
         !!! assertMLSNotEnabled
+
+--------------------------------------------------------------------------------
+-- Mock utilities
```

Comment:
How about moving them to a utilities module?

Reply:
Done.